.github/workflows/ci.yaml

```diff
@@ -29,12 +29,26 @@ jobs:
             -   name: "Install Sphinx dependencies"
                 run: sudo apt-get install python-dev build-essential
 
+            -   name: "Cache pip"
+                uses: actions/cache@v1
+                with:
+                    path: ~/.cache/pip
+                    key: ${{ runner.os }}-pip-${{ hashFiles('_build/.requirements.txt') }}
+                    restore-keys: |
+                        ${{ runner.os }}-pip-
+
             -   name: "Install Sphinx"
                 run: pip install --user sphinx
```

Comment:
@OskarStark Why is this needed? Because `_build/.requirements.txt` has also a reference for sphinx but in version 1.3.6

Reply:
Not sure, let’s try without...
Initially I used the same approach as on Travis

Reply:
It wasn't part of the travis config: https://github.com/symfony/symfony-docs/pull/13061/files#diff-354f30a63fb0907d4ad57269548329e3

So I guess it has something to do with GitHub actions or otherwise, can be removed

Reply:
Hmm strange, maybe it was part of .symfony.cloud.yaml ?

Reply:
I removed it, works fine.